db.json

```diff
@@ -152,7 +152,11 @@
     },
     {
       "id": 200,
+<<<<<<< HEAD
```

Comment:
remove these diff lines..I think you're probably keeping either `changes` or `changelog` line